src/components/ProjectAddSummary/LinkedReleasesAndProjects.tsx

```diff
@@ -11,13 +11,17 @@
 
 import LinkedReleases from './component/LinkedReleasesAndProjects/LinkedReleases'
 import LinkedProjects from './component/LinkedReleasesAndProjects/LinkedProjects'
-// import { Session } from '@/object-types/Session'
+import Session from '@/object-types/Session'
```

Comment:
Nope nope nope

Reply:
@heliocastro Sorry, I didn't get your point. As the `Session.ts` has default export so the import syntax should be like this right ?
Or I'm in other direction ?

Reply:
Was, but now since you followed the refactor process, everything that is global is moving go a declared "unpack" folder,
look on src/object-types/index.ts

And this is defined here in tsconfig.json:
```json
        "paths": {
            "next-sw360": ["./src/components/sw360"],
            "@/app/*": ["./src/app/*"],
            "@/assets/*": ["./src/assets/*"],
            "@/components/*": ["./src/components/*"],
            "@/messages": ["./messages"],
            "@/object-types": ["./src/object-types"],
            "@/object-types/*": ["./src/object-types/*"],
            "@/services/*": ["./src/services/*"],
            "@/styles/*": ["./src/styles/*"],
            "@/utils/*": ["./src/utils/*"],
            "@/utils": ["./src/utils"]
        },
```

So, your changes work now, but will be deprecated in the future for globals.
Now works the two ways, direct like you changed or unpacked:

```json
            "@/object-types": ["./src/object-types"],
            "@/object-types/*": ["./src/object-types/*"],
```

With the time, we will move non global types to their page area, to avoid this huge amount of info there

Reply:
@heliocastro Please review once.